web/src/components/zonemap.jsx

```diff
@@ -126,34 +137,21 @@ const ZoneMap = ({
     }
   }, []); // eslint-disable-line react-hooks/exhaustive-deps
 
-  // TODO: Consider moving the calculation to a useMemo function
-  // change color of zones if timeslider is changed
-  useEffect(() => {
-    if (isLoaded && co2ColorScale) {
-      // TODO: This will only change RENDERED zones, so if you change the time in Europe and zoom out, go to US, it will not be updated!
-      // TODO: Consider using isdragging or similar to update this when new zones are rendered
-      const features = ref.current.queryRenderedFeatures();
```

Comment:
It doesn't make sense to only re-render visible zones as the user might drag. We need to update them all instead